src/components/NavigationListAccordion.tsx

```diff
@@ -36,7 +36,8 @@ export const NavigationListAccordion = ({
       sx={{
         mb: '0.5rem',
         bgcolor: 'secondary.dark',
-      }}>
+      }}
+      onClick={() => window.scrollTo({ top: 0, behavior: 'smooth' })}>
```

Comment:
Finnes et argument for at vi ikke burde gjøre dette når man trykker på elementet man allerede er på, men syns det gir mening å gå til toppen da også, som en slags "reset" og gi en visuell feedback på trykket.